code/modules/food_and_drinks/plate.dm

```diff
@@ -0,0 +1,63 @@
+/obj/item/plate
+	name = "plate"
+	desc = "Holds food, powerful. Good for morale when you're not eating your spaghetti off of a desk."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "plate"
+	w_class = WEIGHT_CLASS_HUGE //No backpack.
```

Comment:
```suggestion
	w_class = WEIGHT_CLASS_BULKY //No backpack.
```
"/// Items that can be weilded or equipped but not stored in an inventory, (e.g. defibrillator, backpack, space suits)
#define WEIGHT_CLASS_BULKY    4
/// Usually represents objects that require two hands to operate, (e.g. shotgun, two-handed melee weapons)
#define WEIGHT_CLASS_HUGE     5"